packages/atomic/src/components/common/generated-answer/generated-content/generated-markdown-content.spec.ts

```diff
@@ -0,0 +1,118 @@
+import DOMPurify from 'dompurify';
+import {html} from 'lit';
+import {afterEach, describe, expect, it, vi} from 'vitest';
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {
+  type GeneratedMarkdownContentProps,
+  renderGeneratedMarkdownContent,
+} from './generated-markdown-content';
+import * as markdownUtils from './markdown-utils';
+
+vi.mock('dompurify');
+
+vi.mock('./markdown-utils', async (importOriginal) => {
+  const actual = await importOriginal<typeof markdownUtils>();
+  return {
+    ...actual,
+    transformMarkdownToHtml: vi.fn(actual.transformMarkdownToHtml),
+  };
+});
```

Comment:
mocking of markdown-utils [2/4]

Spy everything, it's less verbose, and move the reader concerns about the mocked function closer to its usage.

```suggestion
vi.mock('./markdown-utils', {spy: true})
```